Kernel/Bus/USB/Drivers/MassStorage/MassStorageDriver.cpp

```diff
@@ -131,6 +131,60 @@ ErrorOr<void> MassStorageDriver::initialise_bulk_only_device(USB::Device& device
     auto in_pipe = TRY(BulkInPipe::create(device.controller(), device, in_pipe_address, in_max_packet_size));
     auto out_pipe = TRY(BulkOutPipe::create(device.controller(), device, out_pipe_address, out_max_packet_size));
 
+    SCSI::Inquiry inquiry_command {};
+    inquiry_command.allocation_length = sizeof(SCSI::StandardInquiryData);
+
+    SCSI::StandardInquiryData inquiry_data;
+
+    auto inquiry_response = TRY(send_scsi_command<SCSIDataDirection::DataToHost>(*out_pipe, *in_pipe, inquiry_command, &inquiry_data, sizeof(inquiry_data)));
+
+    if (inquiry_response.status != CSWStatus::Passed) {
+        dmesgln("SCSI/BBB: Inquiry failed with code {}", to_underlying(inquiry_response.status));
+    } else {
+        dbgln("SCSI/BBB: Inquiry succeeded");
+        dbgln("    Device Type: {}", inquiry_data.device_type_string());
+        dbgln("    Removable: {}", (inquiry_data.removable & 0x80) == 0x80);
+        dbgln("    Version: {:#02x}", inquiry_data.version);
+        dbgln("    Vendor: {}", StringView { inquiry_data.vendor_id, 8 });
+        dbgln("    Product: {}", StringView { inquiry_data.product_id, 16 });
+        dbgln("    Revision: {}", StringView { inquiry_data.product_revision_level, 4 });
```

Comment:
This output splits up the "USB MassStorage Interface found" text:
```
0.591 [xHCI Controller(8:9)]: USB MassStorage Interface for device 18164:1 found:
0.591 [xHCI Controller(8:9)]:     Subclass: SCSI-transparent [0x06]
0.591 [xHCI Controller(8:9)]:     Protocol: Bulk only [0x50]
0.599 [#0 xHCI Controller(8:9)]: SCSI/BBB: Inquiry succeeded
0.599 [#0 xHCI Controller(8:9)]:     Device Type: Direct Access Block Device
0.599 [#0 xHCI Controller(8:9)]:     Removable: false
0.599 [#0 xHCI Controller(8:9)]:     Version: 0x05
0.599 [#0 xHCI Controller(8:9)]:     Vendor: QEMU    
0.599 [#0 xHCI Controller(8:9)]:     Product: QEMU HARDDISK   
0.599 [#0 xHCI Controller(8:9)]:     Revision: 2.5+
0.599 [xHCI Controller(8:9)]:     Block Size: 512B
0.599 [xHCI Controller(8:9)]:     Block Count: 1811303
0.599 [xHCI Controller(8:9)]:     Total Size: 884MiB
```

(side note: we should print the VID/PID as a 4-digit hex string)

Reply:
Adjusted it to fit in